mempool/clist_mempool.go

```diff
@@ -39,14 +39,15 @@ type CListMempool struct {
 	preCheck  PreCheckFunc
 	postCheck PostCheckFunc
 
-	txs          *clist.CList // concurrent linked-list of good txs
+	txs          *clist.CList // concurrent linked-list of valid txs
```

Comment:
I would keep the original version here. It might not be great, but we are not changing in this PR the semantics of the `txs` list.

Reply:
We can keep the original comment. I thought to change it to keep the wording consistent across the code and the documentation. In fact, I don't think it is anywhere defined what is a "good" or "bad" transaction.

Reply:
The terms good/bad correspond to valid/invalid. Maybe, having a PR to fix this, as well as others inconsistent comments in the  code, is better.

Reply:
Ok, I reverted all these small changes to the comments and logs.